Realm/Realm/DatabaseTypes/RealmValue.cs

```diff
@@ -30,9 +30,10 @@
 namespace Realms
 {
     /// <summary>
-    /// A type that can represent any valid Realm data type. It is a valid type in and of itself,
-    /// which means that it can be used to declare a property of type <see cref="RealmValue"/> that
-    /// can hold any type.
+    /// A type that can represent any valid Realm data type. It is a valid type in and of itself, which
+    /// means that it can be used to declare a property of type <see cref="RealmValue"/>.
+    /// Please note that a <see cref="RealmValue"/> property in a managed <see cref="IRealmObjectBase">realm object</see>
+    /// cannot contain an <see cref="IEmbeddedObject">embedded object</see> nor an <see cref="IAsymmetricObject">asymmetric object</see>.
```

Comment:
Don't want to be a grammar nazi, but I think "or" is the correct conjunction here.

```suggestion
    /// cannot contain an <see cref="IEmbeddedObject">embedded object</see> or an <see cref="IAsymmetricObject">asymmetric object</see>.
```